graphiti_core/search/search_utils.py

```
@@ -331,38 +327,40 @@ async def node_fulltext_search(
+        records, headers, _ = await driver.execute_query(
+            query,
+            params=filter_params,
+            query=fuzzy_query,
+            group_ids=group_ids,
+            limit=limit,
+            database_=DEFAULT_DATABASE,
+            routing_='r',
+        )
+        if driver.provider == 'falkor':
+            headers = [h[1] for h in headers]
+            records = [dict(zip(headers, row)) for row in records]
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)